Xamarin.Forms.Platform.iOS/CollectionView/CarouselViewController.cs

```diff
@@ -182,12 +236,33 @@ void UnsubscribeCollectionItemsSourceChanged(IItemsViewSource oldItemsSource)
 				oldObservableItemsSource.CollectionItemsSourceChanged -= CollectionItemsSourceChanged;
 		}
 
-		void CarouselViewPropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs changedProperty)
+		async void CarouselViewPropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs changedProperty)
```

Comment:
async void?

Reply:
yeah , we can drop this now that we don't handle the change from loop value

Reply:
Fixed @samhouts